app/src/main/java/org/p2p/wallet/auth/ui/generalerror/timer/OnboardingGeneralErrorTimerFragment.kt

```diff
@@ -36,17 +36,20 @@ class OnboardingGeneralErrorTimerFragment :
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
+        requireActivity().onBackPressedDispatcher.addCallback(viewLifecycleOwner) {
+            returnToPhoneNumberEnter()
+        }
         binding.buttonToStartingScreen.setOnClickListener {
             returnToPhoneNumberEnter()
         }
```

Comment:
```suggestion
        binding.buttonToStartingScreen.setOnClickListener {
            returnToPhoneNumberEnter()
        }
        requireActivity().onBackPressedDispatcher.addCallback(viewLifecycleOwner) {
            returnToPhoneNumberEnter()
        }
```
i saw we place backPressedDispatcher always in the end of the onViewCreate